pkg/operator/controllers/condition.go

```diff
@@ -38,29 +43,29 @@ func SetCondition(ctx context.Context, arocli aroclient.Interface, cond *status.
 	})
 }
 
-func setStaticStatus(cluster *arov1alpha1.Cluster, role string) (changed bool) {
-	conditions := make(status.Conditions, 0, len(cluster.Status.Conditions))
+// func setStaticStatus(cluster *arov1alpha1.Cluster, role string) (changed bool) {
```

Comment:
Either leave them or remove. Does this not make condition to flap or show false-positive?